src/routes/mods.rs

```diff
@@ -15,6 +16,45 @@ pub async fn mod_search(
     Ok(HttpResponse::Ok().json(results))
 }
 
+#[derive(Serialize, Deserialize)]
+pub struct ModIds {
+    pub ids: String,
+}
+
+// TODO: Make this return the full mod struct
```

Comment:
This TODO seems to be completed

Reply:
It isn't as the get multiple mods returns part of the mod, not the loaders/categories